support/client/lib/mil-sym/cws.js

```diff
@@ -5230,10 +5242,48 @@ define( function(){
                   "symbolType": "DefinedMissionGraphic", 
                   "defaultControlPts": [ 100,100, 100,0, 0,50  ],
                   "defaultDimensions": [100, 100]
+            },
+            "TACGRP.MOBSU.OBSTBP.CSGSTE.LANE": {
+                  "symbolID": "G*MPBCL---****X",
+                  "tag": "WATER CROSSING LANE",
+                  "symbolType": "SimpleLine"
+            },
+            "TACGRP.MOBSU.OBSTBP.CSGSTE.RFT": {
+                  "symbolID": "G*MPBCR---****X",
+                  "tag": "WATER CROSSING RAFT SITE",
+                  "symbolType": "SimpleLine"
+            },
+            "TACGRP.MOBSU.OBSTBP.CSGSTE.ASTCA": {
+                  "symbolID": "G*MPBCA---****X",
+                  "tag": "WATER CROSSING ASSAULT CROSSING AREA",
```

Comment:
When  I try to create this one and `WATER CROSSING BRIDGE OR GAP`, I get the following error from `renderMissionGraphic` (in `tdg-missionGfxManagement.js` `computeOriginPoint`):

    Warning: symbolID: "GFMPBCA---****X" requires a minimum of 4 points. 3 are present.

Reply:
Ah.  Thanks for pointing that out.  I'll change it to a 4-point simple lane.  Easy fix.

Reply:
The one above it was wrong too - should be a 4-point simple lane.